docs/integrate/mqtt/learn.md

```diff
@@ -0,0 +1,75 @@
+(mqtt-learn)=
+
+# Load data from an MQTT topic into CrateDB
+
+The tutorial will walk you through starting an MQTT broker and CrateDB,
+publishing JSON data to an MQTT topic, subscribing to the topic to relay
+data into a CrateDB table continuously, and validating that the data has
+been stored successfully.
+The data transfer is supported by the [LorryStream MQTT source] data
+pipeline element.
+
+## Prerequisites
+
+Docker is used for running the [Eclipse Mosquitto] MQTT broker and to
+invoke its client commands `mosquitto_pub` and `mosquitto_sub`, as well
+as the data pipeline program `lorry` and the CrateDB shell `crash`.
+This will save you from needing to install them on your workstation,
+but will require a Docker installation.
+
+Alternatively to that, you can also use Podman. Alternatively to Mosquitto, you
+can also use a different MQTT broker like HiveMQ, BunkerM, or even RabbitMQ or
+Azure IoT Hub, because both also talk MQTT.
+
+Start CrateDB.
+```shell
+docker run --name=cratedb --rm -it \
+  --publish=4200:4200 --publish=5432:5432 \
+  --env=CRATE_HEAP_SIZE=2g crate/crate:6.0.0 -Cdiscovery.type=single-node
+```
+
+Start Mosquitto.
+```shell
+docker run --name=mosquitto -it --rm \
+  --publish=1883:1883 eclipse-mosquitto:2.0 \
+  mosquitto -c /mosquitto-no-auth.conf
+```
+
+Prepare shortcuts for the CrateDB shell, LorryStream, and the Mosquitto client
+programs. If you don't use Docker, but installed them on your workstation, just
+skip this section.
+```shell
+alias crash="docker run --rm -it --network=host ghcr.io/crate/cratedb-toolkit:latest crash"
+alias lorry="docker run --rm -i --network=host ghcr.io/daq-tools/lorrystream:latest lorry"
+alias mosquitto_pub="docker run --rm -i --network=host eclipse-mosquitto:2.0 mosquitto_pub"
+alias mosquitto_sub="docker run --rm -i --network=host eclipse-mosquitto:2.0 mosquitto_sub"
+```
```

Comment:
@coderabbitai suggests for `docs/integrate/mqtt/learn.md`:

<blockquote>

`42-46`: **Portability note: Docker host networking.**

--network=host is unsupported on Docker Desktop (macOS/Windows). Consider adding a note with alternatives (user-defined bridge or host.docker.internal).

Example alternatives (outside the selected range):
- Use a bridge network:
  - docker network create mqtt-demo
  - Add --network=mqtt-demo to all docker run aliases and containers
- Or, keep host networking on Linux and document replacing localhost with host.docker.internal in the MQTT/CrateDB URLs on Docker Desktop.

</blockquote>

Reply:
The problem on Windows here are certainly the `alias` commands. If you still like the strategy, we'd need to write down the Windows pendants, when applicable.

When exercising it, please also validate network connectivity with Docker/Windows, to accomodate @coderabbitai suggestions. 🙏

Reply:
@coderabbitai: f6c51e8614f3ff improves the situation further, by both implementing the recommended cross-platform approach for Docker, but also at the spot about the `alias` command, where we are also just following your advises. Thanks a stack!